content/blog/urbit-for-normies.md

```diff
@@ -0,0 +1,181 @@
++++
+title = "Urbit for Normies"
+date = 2020-02-12
+description = "What is Urbit and why should you care?"
+[extra]
+author = "Erik Newton"
+ship = "~patnes-rigtyn"
+image ="https://media.urbit.org/site/posts/essays/normies-2.png"
++++
+
+## What is Urbit and why should you care?
+
+*A layperson’s guide to the coming new internet.*
+
+Urbit is a challenging project to fully describe because it’s an entire system rather than a single product. Often when I talk about one of the individual elements of the system, someone will tell me that another company is already building that — and they’re usually right. What sets Urbit apart is that we’re building all these elements at once to work together seamlessly.
+
+For that reason, this overview is long. I wanted to explain as much of the system at a high level as possible; this is my attempt to share an image of the entire forest rather than spending too much time with any of the trees. 
+
+My own path to Urbit was not predictable, and so it turns out I’m a useful ambassador to describe what the heck we’re doing to my fellow normies.
+
+I refer to myself as a normie because I’m a lawyer by training, not the typical code-savvy early Urbit adopter. I ran a law firm for a decade, and after retiring that, founded a podcast and magazine about relationship dynamics. During that time I came to believe that humanity was in desperate need of better ways to relate to one another. I didn’t know how, but I knew that my podcast about romance wasn’t going to move the needle. I began to look for a bigger project. I wanted something I truly could invest myself in, something that could have a real impact on this problem.
+
+That’s when a friend introduced me to Urbit, saying it was the most credible project he’d seen in years; he described it as “an open-source decentralized internet project,” and he said it “would give control of computing back to regular people.” That definitely sparked my interest, but I didn’t totally understand it. After looking over the website and watching a couple of obscure descriptive videos on YouTube, my response was threefold: 
+
+1) This is huge — if it actually works, it’s the future.
+2) This is real — It’s got working code, dozens of engineers, and a very unorthodox solution.
+3) I don’t truly understand this — I can’t read the code, and I can barely comprehend the paradigm shift.
```

Comment:
"This is huge", "This is real", "I don't truly understand this" should be in bold per GDoc.

Reply:
That's two asterisks — one will italicise, I'm only saying because I mess it up all the time.